### PR TITLE
Add graphic tablet pen pressure/tilt support

### PR DESCRIPTION
**Pull request by bruvzg**
Adds `pressure` and `tilt` properties to `InputEventMouseMotion` event.

Test project: [demo.zip](https://github.com/godotengine/godot/files/2750029/demo.zip), https://streamable.com/5mkgi

screenshot 2019-01-11 at 17 00 55

### CODE REVIEW

**Comment by bruvzg** [comment]
>Any plans for Linux / Windows support?

Added Windows 8+ support, tested on Windows 10, and should build and work on Windows 7 (pen functions are loaded dynamically).

**Comment by bruvzg** [comment]
Added Linux/X11 support.

**Comment by hpvb** [comment]
X11 code looks good

**Comment by reduz** [comment]
Sorry, I missed this PR. I suggest we wait for after 3.1 is out and make it a 3.2 feature at this point. Other than that it looks ok.

**Comment by blurymind** [comment]
This is fantastic! Thank you for implementing it. ANy hope to get android support? Would be amazing for  galaxy note devices

File: platform/windows/os_windows.h [hunk 1]
```diff
@@ -56,6 +56,75 @@
...
+	INT32 tiltY;
+} POINTER_PEN_INFO;
+
+#endif
+
+typedef BOOL(WINAPI *GetPointerTypePtr)(uint32_t p_id, POINTER_INPUT_TYPE *p_type);
+typedef BOOL(WINAPI *GetPointerPenInfoPtr)(uint32_t p_id, POINTER_PEN_INFO *p_pen_info);
+
+/**
+	@author Juan Linietsky <reduzio@gmail.com>
```
Comment (akien-mga) [inline platform/windows/os_windows.h]: You can remove this, it was readded by mistake while solving merge conflicts.

**Comment by akien-mga** [comment]
Too bad I overlooked this over the past few months, it would have been good to merge it earlier to get wider testing.

I guess we can still try to pull it into 3.2 and ask users to test it to see how it goes.

**Comment by akien-mga** [comment]
CC @NathanLovato whom I know has a graphic tablet.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Testing on Linux on my laptop's touchscreen with an HP pen (no tilt support) and I can confirm that pressure support works well :)

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**